gr-utils/python/modtool/templates/gr-newmod/CMakeLists.txt

```diff
@@ -141,9 +141,7 @@ add_subdirectory(apps)
 add_subdirectory(docs)
 add_subdirectory(swig)
 add_subdirectory(python)
-if(ENABLE_GRC)
-    add_subdirectory(grc)
-endif(ENABLE_GRC)
+add_subdirectory(grc)
```

Comment:
yeah, that should be fine. 

We should, however, probably be doing this based on a tiny bits of smarts. For example, conditionalize on the GNU Radio installation actually having GRC enabled. However, our modern CMake Stuff currently doesn't have a GRC target, sooo. Yeah. Don't conditionalize on a variable nobody set.